File: exercism-go/raindrops/raindrops.go
// Package raindrops provides a function that convert raindrops to its appropriate sound.
package raindrops

import "fmt"

// Convert returns a sound based on the number of raindrops.
func Convert(drops int) string {
	s := ""
	if drops%3 == 0 {
		s += "Pling"
	}
	if drops%5 == 0 {
		s += "Plang"
	}
	if drops%7 == 0 {
		s += "Plong"
	}

	if s == "" {
		s = fmt.Sprintf("%v", drops)
	}

	return s
}

File: exercism-python/raindrops/raindrops.py
drops = ((3,'Pling'), (5,'Plang'), (7,'Plong'))

def raindrops(number):
    """
    Converts a number to a string according to the raindrop sounds.
    """

    speak = ''.join([s for n, s in drops if number%n == 0])
    return speak if speak else str(number)
